src/locales/pl-PL.json

```
@@ -116,7 +118,7 @@
   "$clickToZoomEnabled": "Przybliżenie przez kliknięcie myszą",
   "$reducedMotion": "Zmniejsz ruch (wyłącza animacje)",
   "$preserveViewport": "Zachowanie przybliżenia",
-  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer#contributors'>Więcej informacji o Universal Viewer</a>",
+  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer#contributors'>więcej informacji</a>",
```

Just one more line in need of restoration:

```suggestion
  "$uvWebsite": "<a href='https://github.com/universalviewer/universalviewer#contributors'>Więcej informacji o Universal Viewer</a>",
```